docs/content/en/_index.html

```diff
@@ -12,13 +12,20 @@
 </div>
 
 <div class="mx-auto">
-	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
-		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs/quickstart/" >}}">
+		Quick Start <span class="ml-2">🚀</span>
 	</a>
-	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/pipe-cd/pipe">
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="https://github.com/pipe-cd/pipe">
 		GitHub <i class="fab fa-github ml-2 "></i>
 	</a>
 </div>
+
+<div class="mx-auto">
+	<a class="release-panel" target="_blank" href="https://github.com/pipe-cd/pipe/releases">
+		PipeCD <span class="bolder">0.10.2</span> is now available
```

Comment:
`v0.10.2`
It would be nice if this could be fetched automatically.

Reply:
it would. Updating manually is quite tedious.

Reply:
It is, the current commit just because I don't actually know how to get that value. But as for an idea, maybe we could automatically generate it as we do with the release note? 🤔

Reply:
@cakecatz Is there any good way to add a script to load the latest version from this file when the page is loaded?
https://github.com/pipe-cd/pipe/blob/master/release/RELEASE

Reply:
📝  this command (with jq) will fetch the latest tag name
```
curl --silent "https://api.github.com/repos/pipe-cd/pipe/releases/latest" | jq ".tag_name"
```

Reply:
Or to be simple, we can update this CI job to replace that variable before publishing.
https://github.com/pipe-cd/pipe/blob/master/.kapetanios/postsubmit.yaml#L72-L102

Reply:
The idea is:  using Hugo variable in this `_index.html` file. And update our CI task to extracting the version from `RELEASE` file and configuring it when executing this hugo [command](https://github.com/pipe-cd/pipe/blob/master/.kapetanios/postsubmit.yaml#L88).

Reply:
@nghialv 
```js
fetch('https://raw.githubusercontent.com/pipe-cd/pipe/master/release/RELEASE')
  .then(res => res.text())
  .then(console.log)
```

It can get the version with this script, but I think [this idea](https://github.com/pipe-cd/pipe/pull/2141#discussion_r659425863) is better.

Reply:
how about this 👀 
https://github.com/pipe-cd/pipe/pull/2141/commits/885931278bc33fb6345fe7e956e66bd85770ce24

Reply:
I think this is good except for doing a patch release. In that case, the tag and the version number specified in the RELEASE file will not be the same.

Reply:
okay, then I need at least `curl` and `cut` command to process this data like below 👀 
```
╰─ curl --silent https://raw.githubusercontent.com/pipe-cd/pipe/master/release/RELEASE | cut -c9-  
 v0.10.2
 ```

Reply:
we can use "cut" for the local file. The whole repo was already cloned.

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/2141/commits/3f7e6e5663476fb2d90d496624ccce34b3899f1d 🙏